cla_backend/libs/eligibility_calculator/tests/test_calculator.py

```diff
@@ -1926,3 +1933,11 @@ def test_cfe_request_with_applicant_receives_qualifying_benefit(self):
         _, cfe_result = self.checker_with_facts(on_passported_benefits=False)._do_cfe_civil_check()
         self.assertEqual('eligible', cfe_result.overall_result)
         self.assertEqual(False, cfe_result.applicant_details()["receives_qualifying_benefit"])
+
+    def test_cfe_request_with_applicant_receives_asylum_support(self):
+        _, cfe_result = self.checker_with_facts(on_nass_benefits=False)._do_cfe_civil_check()
```

Comment:
why does this come out eligible? I guess due to lack of income - so it probably invalidates the following test (see comment above about proceeding type)

Reply:
Updated test to avoid confusion